src/index.html

```diff
@@ -17,6 +17,61 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="link-logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo company logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a
```

Comment:
Remember about correct indentation between parent and child elements. The `<li>` elements should be indented properly within the `<ul>`.